apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/klienter/GrunnlagKlient.kt

```diff
@@ -54,7 +54,7 @@ class GrunnlagKlientImpl(
         opplysningsbehov: Opplysningsbehov,
     ) {
         return grunnlagHttpClient
-            .post("$url/grunnlag/sak/${opplysningsbehov.sakId}/behandling/$behandlingId/oppdater-grunnlag") {
+            .post("$url/grunnlag/sak/${opplysningsbehov.sakId}/behandling/$behandlingId/opprett-grunnlag") {
```

Comment:
Ifm. revurdering gjøres det uthenting av persongalleri som sendes hit. Dette kan kanskje endres til å bruke `oppdater-grunnlag` nå?

Reply:
Skal ta en titt

Reply:
Ja, men det løses enkelt ved å hente gjeldende persongalleri for sak.

Reply:
Ok, det kan jeg gjøre. Men persongalleriet som sendes inn brukes til noe statistikkgreier også...

Reply:
Konklusjonen ble at vi lar revurdering være, siden den trenger persongalleri til noe annet enn å legge inn nytt grunnlag. Men persongalleri for sak hentes ifm oppdatering av grunnlag gitt eksisterende persongalleri